inference-engine/src/gna_plugin/gna_graph_compiler.cpp

```diff
@@ -501,12 +501,18 @@ void GNAGraphCompiler::PowerPrimitive(InferenceEngine::CNNLayerPtr layer) {
     auto input = layer->insData[0].lock();
 
     auto outputs = *layer->outData.begin();
-
     uint32_t num_rows_in = FROM_IR_DIM(input, 1);
     uint32_t num_columns_in = FROM_IR_DIM(input, 2);
     uint32_t num_rows_out = num_rows_in;
     uint32_t num_padding = ALIGN(num_rows_in, 8) - num_rows_in;
 
+    if (num_columns_in > 8) {
```

Comment:
@mdeisher can we divide by 8, and set num colums to 8 for this diagonal layer?  If this possible  we can save in weights size and also increase performance i guess